llvm/test/CodeGen/AIE/aie2/GlobalISel/post-select-licm-reserved.mir

```diff
@@ -385,3 +385,42 @@ body:             |
     PseudoJNZ %0, %bb.2
     PseudoJ_jump_imm %bb.1
 ...
+---
```

Comment:
Nit: Maybe add a small description, e.g. `Verify that non-reserved physical registers aren't considered for LICM`